Code of conduct revision (#11)
bids-standard/.github: CODE_OF_CONDUCT.md

[Code of Conduct Enforcement Procedure](#code-of-conduct-enforcement-procedure)

[Summary of processes](#summary-of-processes)

[Acknowledge the report](#acknowledge-the-report)

[Evaluating a report](#evaluating-a-report)

[Jurisdiction](#jurisdiction)

[Impact](#impact)

[Risk](#risk)

[Propose a behavioral modification plan](#propose-a-behavioral-modification-plan)

[Propose consequences](#propose-consequences)

[Work group vote](#work-group-vote)

[Administrators/moderators approval](#administrators/moderators-approval)

[Follow up with the reported person](#follow-up-with-the-reported-person)

[Decide further responses](#decide-further-responses)

[Follow up with the reporter](#follow-up-with-the-reporter)

[Documentation and Privacy policies](#documentation-and-privacy-policies)

[Committee online discussion](#committee-online-discussion)

[Shared documentation](#shared-documentation)

[Status Spreadsheet](#status-spreadsheet)

[Report Documentation](#report-documentation)

[Ongoing Documentation of Event and Community Bans](#ongoing-documentation-of-event-and-community-bans)

[Privacy Concerns](#privacy-concerns)

[Reference](#reference)

> ## Code of Conduct {#coc}

Reviewer: Style point: These `{#anchor}`s are being shown but not linked. This does not seem to be a part of GitHub-flavored markdown.

There is an outline available on all Markdown documents by clicking the button in the top right of the page:

![image](https://github.com/user-attachments/assets/8e003f4a-5691-4ecd-bb37-fd84463aefc1)

#### Confidentiality

All reports will be kept confidential. When we discuss incidents with people who are reported, we will anonymize details as much as we can to protect the reporter’s privacy.

However, some incidents happen in one-on-one interactions, and even if the details are anonymized, the reported person may be able to guess who made the report. If you have concerns about retaliation or your personal safety, please note those in your report. We still encourage you to report, so that we can support you while keeping our community members safe. In some cases, we can compile several anonymized reports into a pattern of behavior, and take action on that pattern.

In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.

#### Report Handling Procedure

When you make a report to a member of the Code of Conduct working group, they will gather information about the incident according to the BIDS Procedure For Incident Response.

> After an incident responder takes the report, they will immediately establish a 3 person Code of Conduct Response Group with at least 1 (past or present) BIDS steering group members and 1 BIDS maintainer, unless there is a conflict of interest, in which case any non-interested parties will be contacted.

Reviewer: Reads oddly to me. Maybe this?

```suggestion
After an incident responder takes the report, they will immediately establish a 3 person Code of Conduct Response Group with at least 1 (past or present) BIDS steering group members and 1 BIDS maintainer, unless conflicts of interest prevent such an arrangement, in which case any disinterested party may be recruited.
```

If the incident is ongoing and needs to be immediately addressed, any of the members of the 3-person CoC response group may take appropriate action to ensure the safety of everyone involved. If the situation requires it, this may take the form of a referral to an appropriate non-BIDS entity, including local law enforcement. The BIDS community is not equipped to handle emergency situations.

If the incident is less urgent, the report will be discussed by the response group, which will meet to determine an appropriate response. Examples of possible incident responses are outlined in the BIDS Procedure For Incident Response.

Procedures for in-person gatherings at e.g., conferences and workshops are likely handled by the local organizing committee to discuss all incidents and determine any necessary follow-up actions. Incidents can additionally fall within the scope of the BIDS CoC (e.g., if an incident occurs while a member of the community is speaking as a representative of BIDS).

#### Following up with reporters

> Within one week of an incident report, a member of the response group, will follow up with the person who made the report and provided their contact information. The follow up may include:

Reviewer: ```suggestion
Within one week of an incident report, a member of the response group will follow up with the reporter. The follow up may include:
```

* An acknowledgment that the Code of Conduct responders discussed the situation.
* Whether or not the report was determined to be a violation of the Code of Conduct.
* What actions (if any) were taken to address the reported behavior.

In some cases, the lead responders may need to ask additional questions about the incident in order to identify the reported person.

#### Conflict of Interest

> If an member of the response group has a conflict of interest for a report, they will recuse themselves from the discussion and handling of the incident. The incident documentation will not be available to them, and they will excuse themselves from any conversations involving handling the incident.

Reviewer: ```suggestion
If a member of the working group has a conflict of interest for a report, they will recuse themselves from the discussion and handling of the incident. The incident documentation will not be available to them, and they will excuse themselves from any conversations involving handling the incident.
```

* Nothing, if the behavior was determined to not be a Code of Conduct violation.
* A verbal or emailed warning.
* A final warning.
* Temporarily removing the reported person from the online community.
* Permanently removing the reported person from the online community.
* Publishing an account of the incident.

### Response group vote

Decisions on the behavioral modification plans and consequences will be determined by a majority vote of the BIDS Code of Conduct response group.

### Administrators/moderators approval

Once the response group has approved the behavioral modification plans and consequences, they will communicate the recommended response to the BIDS maintainers. The response group does not need to state who reported this incident. They should attempt to anonymize any identifying information from the report.

The BIDS CoC Working Group will be required to respond back with whether they accept the recommended response to the report. If they disagree with the recommended response, they should provide a detailed response or additional context as to why they disagree. The Working Group is encouraged to respond within a week.

> In cases where the Working Group disagrees on the suggested resolution for a report, the BIDS Code of Conduct work group may choose to notify the BIDS steering group.

Reviewer: The working group includes the steering group.

### Follow up with the reported person

> The BIDS Code of Conduct work group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:

Reviewer: ```suggestion
The BIDS Code of Conduct response group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:
```

* A description of the person's behavior in neutral language
* The negative impact of that behavior
* A concrete behavioral modification plan
* Any consequences of their behavior

The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the work group can accept the apology on behalf of the reporter.

### Decide further responses

> If the reported person provides additional context, the BIDS Code of Conduct work group may need to re-evaluate the behavioral modification plan and consequences.

Reviewer: ```suggestion
If the reported person provides additional context, the BIDS Code of Conduct response group may need to re-evaluate the behavioral modification plan and consequences.
```

### Follow up with the reporter

> A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.

Reviewer: ```suggestion
A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the response group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```

### Follow up with the reported person {#follow-up-with-the-reported-person}

The BIDS Code of Conduct work group will work with online community administrators/moderators to draft a response to the reported person. The email should contain:

* A description of the person's behavior in neutral language
* The negative impact of that behavior
* A concrete behavioral modification plan
* Any consequences of their behavior

The work group should not state who reported this incident. They should attempt to anonymize any identifying information from the report. The reported person should be discouraged from contacting the reporter to discuss the report. If they wish to apologize to the reporter, the work group can accept the apology on behalf of the reporter.

### Decide further responses {#decide-further-responses}

If the reported person provides additional context, the BIDS Code of Conduct work group may need to re-evaluate the behavioral modification plan and consequences.

### Follow up with the reporter {#follow-up-with-the-reporter}

> A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are not made in good faith (such as "reverse sexism" or "reverse racism") may receive no response.

Reviewer: ```suggestion
A person who makes a report should receive a follow-up email stating what action was taken in response to the report. If the work group decided no response was needed, they should provide an email explaining why it was not a Code of Conduct violation. Reports that are determined not to have been made in good faith may receive no response.
```

I really don't think it's a good idea to pick two particular positions to represent bad faith reports. These phrases have particular meanings in American political discourse, but that meaning may not be shared by people of all international backgrounds or people who have not had their ear to the ground in the last decade or so.

I think stating that the group has discretion to determine bad faith or disparate impact ("Impact" section above) is sufficient to cover these cases without inviting debate over what these terms mean, which debate could itself turn into CoC violations.
